hugolib/menu.go

```diff
@@ -213,3 +217,12 @@ func (m Menu) Reverse() Menu {
 
 	return m
 }
+
+func (m *MenuEntry) Title() string {
+	if m.title == "" {
```

Comment:
This is a little bit matter of taste, but I always prefer to return early if I can, and it makes the code less nested and easier to read, .e.g

```go
if m.title != "" {
   return m.title
}

...
```

Reply:
fixed